AntennaTracker/AntennaTracker.cpp

```diff
@@ -144,7 +144,6 @@ const AP_HAL::HAL& hal = AP_HAL::get_HAL();
 Tracker::Tracker(void)
     : logger(g.log_bitmask)
 {
-    memset(&vehicle, 0, sizeof(vehicle));
```

Comment:
We didn't need this in the first place. It was just an attempt at suppressing the coverity warning, and we aren't generally worrying about that.

Reply:
ok